### DISCUSSION TITLE
Actualizacion de cotizacion de presupuestos.

### DISCUSSION THREAD

**Discussion by Blacks-Lab** [OdooAR/odoo-argentina-doc#4] [Q&A] [answered]
Buenas chicos, estaba buscando sugerencias, nose como hacer que luego de actualizar cotizacion de moneda, se actualicen los presupuestos ya sea por acciones planificadas o algo asi o quizas ya hay algun modulo. Gracias.

PD :  V13 CE

**Comment by ibuioli** [comment, marked as answer]
Si entiendo bien, creo que esto te puede servir, es para la 12 pero recuerdo instalarla en la 13:

https://github.com/OCA/sale-workflow/tree/12.0/sale_order_price_recalculation

Agrega un botón que recalcula los precios, ya sea porque cambiaste la lista de precios o porque actualizaste dichos precios. Y si querés automatizarlo más, podrías crear la acción planificada que dispare la acción cada X tiempo, siempre y cuando el presupuesto se encuentre en borrador.

**Reply by Blacks-Lab** [reply]
ya lo pruebo en unas horas muchas gracias.

**Comment by 3lk0k0** [comment]
el presupuesto pdf es generico o hay una plantilla como el de factura electronica ?
